citrination_client/client.py

```diff
@@ -67,6 +67,32 @@ def upload_file(self, file_path, data_set_id):
         else:
             return None
 
+    def show_latest_dataset_files(self, dataset_id):
+        return self._get_content_from_url(self.api_url + '/data_sets/' + str(dataset_id) + '/latest')
+
+    def show_dataset_files(self, dataset_id):
+        return self._get_content_from_url(self.api_url + '/data_sets/' + str(dataset_id) + '/files')
+
+    def show_latest_dataset_file(self, dataset_id, file_path):
```

Comment:
For each of these can you do the following:

- Change "show_..." to "get_..." - I think that's more descriptive of the fact that you are retrieving the record.
- Combine e.g. "show_pif_version" and "show_pif_latest" into a single method "get_pif(self, dataset_id, uid, version=None)". A few important things here... (1) uid rather than pidf_uuid is more consistent with the labels that users use on upload (2) version = None should retrieve the latest version, version != None should get the specified version. Do this same thing for all methods that have a latest and versioned type.